src/System.Runtime.CompilerServices.Unsafe/tests/UnsafeTests.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Runtime.CompilerServices
```

Comment:
I'm not sure we have a guideline for this, I know it's a bit inconsistent elsewhere in the repo. I sort of prefer leaving off "Tests" since the class names almost always have "Tests" in them, but that might just be me.